.github/workflows/build.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       - name: Build and install
         run: rm -rf .eggs && pip install -e .
       - name: Validate the installation
-        run: python -c "import mmcv"
+        run: python -c "import mmcv; print(f'GCC {mmcv.version.gcc_version}', f'CUDA {mmcv.version.cuda_version}', f'PyTorch {mmcv.version.torch_version}', f'CommitID {mmcv.version.commit_id}', sep='\n')"
```

Comment:
we can directly call collect_env.py for simplicity

Reply:
We run this command just to check whether the version info we get is correct on different OS.